src/functions/Funcs/info/dbPing.js

```diff
@@ -1,7 +1,8 @@
 module.exports = async d => {
     const {code} = d.util.openFunc(d);
+    const ping = d.client.db.roundTrip ? await d.client.db.roundTrip() : d.client.db.ping
```

Comment:
A bit unethical.

Reply:
Is this okay? 4d2b95f6e468012938e22eadcc13b2da7a1af3cd